projects/aca-playwright-shared/src/page-objects/components/dataTable/data-table.component.ts

```diff
@@ -246,14 +246,14 @@ export class DataTableComponent extends BaseComponent {
     }
   }
 
-  async selectMultiItem(name: string, name2: string): Promise<void> {
+  async selectMultiItem(...names: string[]): Promise<void> {
     await this.page.keyboard.down('Meta');
-    let row = this.getRowByName(name);
-    await row.locator('[title="Size"]').click();
-    await row.locator('.adf-datatable-checkbox .mat-checkbox-checked').waitFor({ state: 'attached' });
-    row = this.getRowByName(name2);
-    await row.locator('[title="Size"]').click();
-    await row.locator('.adf-datatable-checkbox .mat-checkbox-checked').waitFor({ state: 'attached' });
+    for (const name of names) {
+      let row = this.getRowByName(name);
+      await row.locator('[title="Size"]').click();
+      await row.locator('.adf-datatable-selected').waitFor({ state: 'attached' });
```

Comment:
This locator won't work anymore, there has been a PR to enable multiselection that changes this behaviour for reference please check: https://github.com/Alfresco/alfresco-content-app/pull/3873

Reply:
So you can select multiple files with the checkboxes on the left now? If yes, then I have to update tests with that :D

Also, the shift/command + click will stop working in the near future?

Reply:
Yes this change was requested by our UX designers since it will be required for Legal hold, yes shift/command + click has been replaced by checkboxes for now as far as I can see